optimum/onnxruntime/configuration.py

```diff
@@ -692,6 +705,13 @@ class OptimizationConfig:
     disable_embed_layer_norm: bool = True
     disable_shape_inference: bool = False
 
+    # ONNX Runtime 1.14.0 arguments
+    use_multi_head_attention = False
+    enable_gemm_fast_gelu = False
+    use_raw_attention_mask = False
```

Comment:
So it's different than `use_mask_index` right?

Reply:
Feels a bit redundant to me: https://github.com/microsoft/onnxruntime/blob/6ccaeddefa65ccac402a47fa4d9cad8229794bb2/onnxruntime/python/tools/transformers/fusion_options.py#L92-L95